exposed-core/src/main/kotlin/org/jetbrains/exposed/sql/SQLExpressionBuilder.kt

```diff
@@ -393,7 +393,18 @@ interface ISqlExpressionBuilder {
     /** Checks if this [EntityID] expression is less than some [t] value. */
     @JvmName("lessEntityID")
     infix fun <T : Comparable<T>> Column<EntityID<T>>.less(t: T): LessOp =
-        LessOp(this, wrap(EntityID(t, (this.foreignKey?.targetTable ?: this.table) as IdTable<T>)))
+        LessOp(
```

Comment:
Just to reduce some local code duplication it could be extracted into a separate function. Something like:

```kotlin
@JvmName("lessEntityID")
infix fun <T : Comparable<T>> Column<EntityID<T>>.less(t: T): LessOp = LessOp(this, wrap(EntityID(t, this.originalTable())))

private fun <T : Comparable<T>> Column<EntityID<T>>.originalTable(): IdTable<T> {
    return when (val table = this.foreignKey?.targetTable ?: this.table) {
        is Alias<*> -> table.delegate
        else -> table
    } as  IdTable<T>
}
```